trytond/application.py

```diff
@@ -45,4 +50,31 @@
     for thread in threads:
         thread.join()
 
+
+if uwsgidecorators is not None:
+    # When running under uwsgi, the behaviour will be to the application
```

Comment:
the behaviour will be to the application process ? what do you mean ?

Reply:
Typo, c'est fix